### PR TITLE
Changes after testing

### PR DESCRIPTION
**Pull request by revit13**

### CODE REVIEW

File: charts/fybrik/values.yaml [hunk 1]
```diff
@@ -50,6 +50,24 @@ global:
   # zerolog verbosity level 
   # ref: https://github.com/rs/zerolog#leveled-logging
   loggingVerbosity: -1
+  # If deploying to OpenShift
```
Comment (roytman) [inline charts/fybrik/values.yaml]: maybe "is the target deployment OpenShift"

File: charts/fybrik/values.yaml [hunk 2]
```diff
@@ -160,19 +178,10 @@ manager:
 
   podAnnotations: {}
 
-  podSecurityContext:
-    runAsNonRoot: true
+  podSecurityContext: {}
```
Comment (roytman) [inline charts/fybrik/values.yaml:186]: please add a comment that this setting overwrites the global podSecurityContext settings

File: charts/fybrik/values.yaml [hunk 3]
```diff
@@ -160,19 +178,10 @@ manager:
...
-     runAsUser: 10001
-     seccompProfile:
-       type: RuntimeDefault
-     readOnlyRootFilesystem: true
-     privileged: false
-     allowPrivilegeEscalation: false
-     capabilities:
-      drop:
-      - ALL
+  containerSecurityContext:
```
Comment (roytman) [inline charts/fybrik/values.yaml:191]: a comment about overwriting

File: charts/fybrik/values.yaml [hunk 4]
```diff
@@ -50,6 +50,24 @@ global:
   # zerolog verbosity level 
   # ref: https://github.com/rs/zerolog#leveled-logging
   loggingVerbosity: -1
+  # If deploying to OpenShift
+  openshift: false
+  # Pod Security Context
+  # ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
```
Comment (roytman) [inline charts/fybrik/values.yaml]: please add a comment that this is the default setting for all pods, and can be overwritten by a specific pod settings
maybe use this link https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#security-context

File: charts/fybrik/values.yaml [hunk 5]
```diff
@@ -50,6 +50,24 @@ global:
...
+  podSecurityContext:
+    runAsNonRoot: true
+    runAsUser: 10001
+    seccompProfile:
+      type: RuntimeDefault
+  # Container Security Context to be set on the controller component container
+  # ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
```
Comment (roytman) [inline charts/fybrik/values.yaml]: comment about overwriting 
maybe provide a link https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#security-context

File: charts/fybrik/values.yaml (hunk 1 above)
Comment (revit13) [reply charts/fybrik/values.yaml]: Fixed. Thanks

File: charts/fybrik/values.yaml (hunk 2 above)
Comment (revit13) [reply charts/fybrik/values.yaml:186]: Fixed. Thanks

File: charts/fybrik/values.yaml (hunk 3 above)
Comment (revit13) [reply charts/fybrik/values.yaml:191]: Fixed. Thanks

File: charts/fybrik/values.yaml (hunk 4 above)
Comment (revit13) [reply charts/fybrik/values.yaml]: Fixed. Thanks

File: charts/fybrik/values.yaml (hunk 5 above)
Comment (revit13) [reply charts/fybrik/values.yaml]: Fixed. Thanks

**Review by roytman — APPROVED** [review]

**Pull request merged**